less/components/nav.less

```diff
@@ -23,6 +23,20 @@
         opacity: 1;
       }
     }
+    .nav-lang-selector {
+      position: absolute;
```

Comment:
So I think you can remove the position, right, top, and static height 65 on select.